media-libs/assimp/files/assimp-5.2.5-float-comparison.patch

```diff
@@ -0,0 +1,84 @@
+diff --git a/include/assimp/matrix3x3.inl b/include/assimp/matrix3x3.inl
```

Comment:
When you come back to this, please add some explanation to the top. Thanks!

Reply:
Bit-wise floating point number comparison is down right criminal... It's also a wild mix in there so I just use the operators with tolerances.